thumbor_botornado/s3_http_loader.py

```diff
@@ -0,0 +1,9 @@
+import thumbor_botornado.s3_loader as S3Loader
+import thumbor.loaders.http_loader as HttpLoader
+import re
+
+def load(context, url, callback):
+	if re.match('https?:', url, re.IGNORECASE):
```

Comment:
You could compile this regex for a slight performance gain:
```python
is_http = re.compile('https?:')
if is_http.match(url, re. IGNORECASE):
```
or something like it